.github/workflows/ci.yml

```diff
@@ -419,68 +421,51 @@ jobs:
         with:
           job-id: remote-infinispan-integration-tests
 
-  store-integration-tests:
-    name: Store IT
+  auroradb-integration-tests:
+    name: AuroraDB IT
     needs: [build, conditional]
-    if: needs.conditional.outputs.ci-store == 'true'
+    if: needs.conditional.outputs.ci-aurora == 'true'
     runs-on: ubuntu-latest
     timeout-minutes: 150
-    strategy:
-      matrix:
-        db: ${{ fromJson(needs.conditional.outputs.ci-store-matrix) }}
-      fail-fast: false
     steps:
       - uses: actions/checkout@v4
 
       - id: aurora-init
         name: Initialize Aurora environment
-        if: ${{ matrix.db == 'aurora-postgres' }}
         run: |
           AWS_REGION=us-east-1
           echo "Region: ${AWS_REGION}"
-
+          
           aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws configure set aws_secret_access_key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws configure set region ${AWS_REGION}
           PASS=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 13; echo)
           echo "::add-mask::${PASS}"
-
-          echo "name=gh-action-$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+          
+          echo "name=gh-action-$(git rev-parse --short HEAD)-${{ github.run_id }}-${{ github.run_attempt }}" >> $GITHUB_OUTPUT
           echo "password=${PASS}" >> $GITHUB_OUTPUT
           echo "region=${AWS_REGION}" >> $GITHUB_OUTPUT
 
       - id: aurora-create
         name: Create Aurora DB
-        if: ${{ matrix.db == 'aurora-postgres' }}
         uses: ./.github/actions/aurora-create-database
         with:
           name: ${{ steps.aurora-init.outputs.name }}
           password: ${{ steps.aurora-init.outputs.password }}
           region: ${{ steps.aurora-init.outputs.region }}
 
-      - id: integration-test-setup
-        name: Integration test setup
-        if: ${{ matrix.db != 'aurora-postgres' }}
-        uses: ./.github/actions/integration-test-setup
-
-      - name: Run Aurora tests on EC2
-        id: aurora-tests
-        if: ${{ matrix.db == 'aurora-postgres' }}
+      - id: ec2-create
+        name: Create EC2 runner instance
         run: |
-          PROPS="-Dauth.server.db.host=${{ steps.aurora-create.outputs.endpoint }}"
-          PROPS+=" -Dkeycloak.connectionsJpa.password=${{ steps.aurora-init.outputs.password }}"
-
           REGION=${{ steps.aurora-init.outputs.region }}
 
           curl --fail-with-body https://truststore.pki.rds.amazonaws.com/${REGION}/${REGION}-bundle.pem -o aws.pem
-          PROPS+=" -Dkeycloak.connectionsJpa.jdbcParameters=\"?ssl=true&sslmode=verify-ca&sslrootcert=/opt/keycloak/aws.pem\""
-
-          TESTS=`testsuite/integration-arquillian/tests/base/testsuites/suite.sh database`
-          echo "Tests: $TESTS"
-
+          PROPS="-Dkeycloak.connectionsJpa.jdbcParameters=\"?ssl=true&sslmode=verify-ca&sslrootcert=/opt/keycloak/aws.pem\""
+          echo "maven_properties=${PROPS}" >> $GITHUB_OUTPUT
```

Comment:
As the properties are related to aurora, not ec2, I think it makes sense for the maven_properties output to be set in the `aurora-init` step. For the same reason, we can also move the `curl` command to the `aurora-init` step.

Reply:
this should be fixed now.